src/main/java/frc/robot/Robot.java

```diff
@@ -171,6 +172,34 @@ public class PDHValues {
     Current RADIO_ = Amps.of(PDH.getCurrent(22));
     Current PORT23 = Amps.of(PDH.getCurrent(23));
```

Comment:
Can you convert these to MutCurrent and MutVoltage? This will save on memory:
https://docs.wpilib.org/en/stable/docs/software/basic-programming/java-units.html#mutability-and-object-creation

Example
```java
MutVoltage voltage = Volts.mutable((PDH.getVoltage()));
...
public void updateValues() {
VOLTAGE.mut_replace(PDH.getVoltage(), Volts)
```